app/controllers/users_controller.rb

```diff
@@ -1,9 +1,12 @@
 class UsersController < ApplicationController
   def index
-    # @users = User.all
+    @users = User.all
   end
 
   def show
-    # @user = User.find(params[:id])
+    @user = User.find(params[:id])
+    @posts = Post.all
+    @comments = Comment.where(post_id: 2)
+    @likes = Like.all
   end
```

Comment:
- [ ] Please kindly take note that I was unable to access the user#show page, I got the error message screenshot below and noticed that the url is not properly implemented. ℹ️ 

<img width="859" alt="Untitled" src="https://github.com/AlaaAlsalem/blogaapp/assets/103386183/31f976fa-8f54-41f1-b554-7a1eee28f3ca">

**Note: I was able to access this page by typing it manually** 🙏